operations/observability/mixins/IDE/rules/components/openvsx-proxy/alerts.libsonnet

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+ */
+
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gitpod-component-openvsx-proxy-alerts',
+        rules: [
+          {
+            alert: 'GitpodOpenVSXRegistryDown',
+            labels: {
+              severity: 'critical',
+            },
+            'for': '20m',
```

Comment:
Do you think it is necessary? It seem in other alerts we don't do it. Would it be good to know it immediately? if 0.01 is too noisy we can increase it?

Reply:
> Do you think it is necessary? It seem in other alerts we don't do it.

Do you mean to decrease this value, I think 20 min is fine to avoid any false positives alerts like [this one](https://grafana.gitpod.io/d/HNOvmGpxgd/openvsx-proxy?orgId=1&from=1658949649687&to=1658955872665)
I don't see the benefit to have an alert immediately if the only thing we can do is just tell eclipse folks to take a look and cross fingers

Reply:
I thought [5m] interval already counts it in? But better to ask @ArthurSens

Reply:
[5m] Is the amount of data that `rate()` will use to calculate the increase rate.

It can be that we see a huge spike that makes a rate over 5minutes to be higher than 0.01, even though it was only for 30 seconds.

The for in the alert makes sure that the rate was continuously higher than 0.01 for the entire duration, so will avoid alerts during strong but short spikes